Results_Engine/Results_Engine.csproj

```diff
@@ -37,12 +37,24 @@
       <HintPath>C:\ProgramData\BHoM\Assemblies\BHoM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Dimensional_oM, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null" />
+    <Reference Include="Geometry_oM">
+      <HintPath>..\..\..\..\..\..\ProgramData\BHoM\Assemblies\Geometry_oM.dll</HintPath>
+    </Reference>
```

Comment:
All the BHoM refs here should be C:\ProgramData\BHoM\Assemblies\....
And copy local set as false 
Also no specific version, culture or public key
The Structure_oM reference is correct.

Reply:
That should be fixed now.